### PR TITLE
Quartz-beta-dev build fixes 

### PR DESCRIPTION
**Pull request by fallsdownhill**
Tested everything against build 1213

### CODE REVIEW

File: feature-layer-selection/src/main/java/com/esri/arcgisruntime/samples/featurelayerselection/MainActivity.java [hunk 1]
```diff
@@ -77,7 +77,6 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
                 Envelope envelope = new Envelope(clickPoint.getX() - mapTolerance, clickPoint.getY() - mapTolerance, clickPoint.getX() + mapTolerance, clickPoint.getY() + mapTolerance, map.getSpatialReference());
                 QueryParameters query = new QueryParameters();
                 query.setGeometry(envelope);
-                query.getOutFields().add("*");
```
Comment (shellygill) [inline feature-layer-selection/src/main/java/com/esri/arcgisruntime/samples/featurelayerselection/MainActivity.java]: This change is fine - I find a more fundamental problem that stops the same working correctly, but that's nothing to do with your change, so I will submit a separate issue for that.

File: change-feature-layer-renderer/src/main/java/com/esri/arcgisruntime/samples/changefeaturelayerrenderer/MainActivity.java [hunk 2]
```diff
@@ -94,21 +94,20 @@ private void createBottomToolbar() {
...
-                            // change the text to reset
-                            overrideActive = true;
-                            item.setTitle(R.string.action_reset);
-                        } else {
-                            resetRenderer();
-                            // change the text to override
-                            overrideActive = false;
-                            item.setTitle(R.string.action_override_rend);
-                        }
+                int itemId = item.getItemId();
```
Comment (shellygill) [inline change-feature-layer-renderer/src/main/java/com/esri/arcgisruntime/samples/changefeaturelayerrenderer/MainActivity.java]: Can you add the comment line about why this uses if-else, like you did for the others? Worth adding these for when people go to update the code.

File: build.gradle [hunk 3]
```diff
@@ -19,8 +19,8 @@ allprojects {
 }
 
 ext {
-    compileSdkVersion = 22
-    buildToolsVersion = "22.0.1"
+    compileSdkVersion = 23
+    buildToolsVersion = "23.0.1"
```
Comment (shellygill) [inline build.gradle]: Upgrading these was necessary I see, due to a compile error if left as previous values. I would say this is OK to go ahead with for now, but will want to make sure @doneill is aware of the change.

**Comment by shellygill** [comment]
Thanks @fallsdownhill - only one change suggestion from me, to add a comment. 

I see this includes the changes from your previous PR - were you planning to ditch that other PR and just merge this one?

**Comment by fallsdownhill** [comment]
@shellygill added the if-statement comments. This pull request has the other changes because I needed a branch to use as the source for the SampleViewer build. I can close that one and just use this pull.

**Comment by shellygill** [comment]
OK, last change looks fine so you can go ahead and merge this.

**Comment by fallsdownhill** [comment]
@shellygill I don't have write permissions for this repository

**Comment by shellygill** [comment]
@fallsdownhill - ah, didnt realise - OK, I will merge it now.

**Pull request merged**